src/modules/todo/api/index.ts

```diff
@@ -0,0 +1,14 @@
+import { TodoItem } from '../components/todo/Todo.interface'
```

Comment:
In that case it would be better to put `ToItem` inside `modules/todo/types` as it's more like general type. Also it would be preferrable to use global import instead of relative.